handlers/epoch.go

```diff
@@ -156,10 +155,15 @@ func Epoch(w http.ResponseWriter, r *http.Request) {
 		http.Error(w, "Internal server error", http.StatusInternalServerError)
 		return
 	}
-	err = db.ReaderDb.Get(&epochPageData.PreviousEpoch, "SELECT epoch FROM epochs WHERE epoch < $1 ORDER BY epoch DESC LIMIT 1", epochPageData.Epoch)
-	if err != nil {
-		logger.Errorf("error retrieving previous epoch for epoch %v: %v", epochPageData.Epoch, err)
+
+	if epochPageData.Epoch == 0 {
```

Comment:
`if epochPageData.Epoch <= 1`

previous epoch is also 0 if the current epoch is 1